public/src/cards.js

```diff
@@ -95,6 +95,8 @@ let events = {
       pack[name] = 1
     }
     App.update()
+    if (document.hidden && App.state.beep)
```

Comment:
beeping is binary; a user could have the tab open but be focused on another window. there needs to be a setting to support beeping only while the tab is hidden